src/tabs/discover/SearchFeed/index.js

```diff
@@ -48,31 +37,17 @@ const SearchFeed = withNavigation(({ navigation, searchText }) => (
   >
     {({ loading, error, data, refetch }) => (
       <StyledFeedView
+        ListItemComponent={({ ...props }) => (
+          <SearchCardConnected Component={FeaturedCard} {...props} />
+        )}
         content={get(data, 'search.edges', [])}
         ListEmptyComponent={() => <NoResults searchText={searchText} />}
         hasContent={get(data, 'search.edges', []).length}
         isLoading={loading}
         error={error}
         refetch={refetch}
+        onPressItem={(item) => handleOnPress({ navigation, item })}
```

Comment:
Be careful with inline functions like this. JavaScript will create space in memory each time it sees an anonymous function. Because this is only one time, it isn't a big deal, but in a list of items, it could become taxing.

Reply:
good to know